android/src/main/java/com/thebluealliance/androidclient/helpers/MatchHelper.java

```diff
@@ -95,7 +96,7 @@ public static TYPE fromKey(String key) {
         SHORT_TYPES.put(MatchHelper.TYPE.SEMI, "sf");
         SHORT_TYPES.put(MatchHelper.TYPE.FINAL, "f");
 
-        LONG_TYPES = new HashMap<>();
+        LONG_TYPES = new HashMap<>(); // TODO: I18N
```

Comment:
`I18N` is short for "internationalization" ("i" followed by 18 letters followed by "n"). It's goofy but common practice, along with `L10N` for "localization".